### PR TITLE
Small tidy-up: Rename test file for databaseFetcher tests

### PR DESCRIPTION
**Pull request by elenatanasoiu**
These are not testing anything in the databases.ts file as we're invoking functions from the DatabaseFetcher.

Let's name the file accordingly.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/test/vscode-tests/cli-integration/databaseFetcher.test.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  promptImportInternetDatabase,
+} from "../../../src/databaseFetcher";
+import { cleanDatabases, dbLoc, DB_URL, storagePath } from "./global.helper";
+
+jest.setTimeout(60_000);
+
+/**
+ * Run various integration tests for databases
+ */
+describe("DatabaseFetcher", () => {
```
Comment (shati-patel) [inline extensions/ql-vscode/test/vscode-tests/cli-integration/databaseFetcher.test.ts:18]: Good idea to move these tests 👍🏽 
Shall we move this commit to a separate PR, so we can get this merged without cluttering up our "actual" change?

Comment (elenatanasoiu) [reply extensions/ql-vscode/test/vscode-tests/cli-integration/databaseFetcher.test.ts:18]: Can do!

**Review by shati-patel — APPROVED** [review]
Makes sense! Thanks for the tidy-up 🧹
